doc/source/serve/monitoring.md

```diff
@@ -317,6 +317,10 @@ The following metrics are exposed by Ray Serve:
        * error_code
        * method
      - The number of non-200 HTTP responses.
+   * - ``serve_num_ongoing_requests`` [*]
```

Comment:
how about changing it to `serve_num_ongoing_http_requests`? since we will have gRPC proxy very soon.

Reply:
Good idea, I renamed it.